packages/components/src/Link/Link.stories.tsx

```diff
@@ -46,11 +53,29 @@ const defaultArgs = {
   href: "",
 };
 
-export const StandardLink = Template.bind(null);
-StandardLink.args = {
+export const Default = Template.bind(null);
+Default.args = {
   ...defaultArgs,
 };
 
+export const PrimaryRecommendedVariants = () =>
+  getStandardSet(Link, "Link", "flat");
+
+export const SecondaryRecommendedVariants = () =>
+  getStandardSet(Link, "Link", "outline");
+
+export const TertiaryRecommendedVariants = () =>
+  getStandardSet(Link, "Link", "outline", "neutral");
+
+export const PlainRecommendedVariants = () =>
+  getPlainRecommendedVariants(Link, "Link");
+
+export const LinkRecommendedVariants = () =>
+  getLinkRecommendedVariants(Link, "Link");
+
+export const DestructiveRecommendedVariants = () =>
+  getDestructiveRecommendedVariants(Link, "Link");
+
```

Comment:
let's keep `linkInBody` & `linkInHeading` above these, think it's important to illustrate that Link inherits its size

Reply:
Moved – thanks!